src/main/java/ai/rev/speechtotext/models/streaming/SessionConfig.java

```diff
@@ -83,8 +84,9 @@ public Boolean getRemoveDisfluencies() {
    * the option to remove disfluencies is optional.
    *
    * @param removeDisfluencies The option to filter profanity.
-   * @see <a
-   *     href="https://www.rev.ai/docs/streaming#section/WebSocket-Endpoint/Remove-Disfluencies-(Beta)">https://www.rev.ai/docs/streaming#section/WebSocket-Endpoint/Remove-Disfluencies-(Beta)</a>
+   * @see <a href="https://www.rev.ai/docs/streaming#section/WebSocket-Endpoint/Remove-Disfluencies-(Beta)">
```

Comment:
This link doesn't work. The correct one is https://www.rev.ai/docs/streaming#section/WebSocket-Endpoint/Remove-Disfluencies